### PR TITLE
OEL-160: Drupal 9 compatibility fixes.

### PR DESCRIPTION
**Pull request by saidatom**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,19 +6,22 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core-recommended": "^8.9 || ^9 || ^10"
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
        "drupal/core": "^8.9 || ^9"
```
we don't guarantee support of Drupal 10

File: composer.json [hunk 2]
```diff
@@ -6,19 +6,22 @@
...
-        "drupal/drupal-extension": "~4.1",
-        "drush/drush": "^10.3",
-        "openeuropa/code-review": "~1.0",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6 || ~7"
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9 || ^10",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "^10.4",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
        "drush/drush": "^10.3",
```
it is minimum supported version

File: composer.json [hunk 3]
```diff
@@ -6,19 +6,22 @@
...
+        "drupal/upgrade_status": "^3.4",
+        "mglaman/drupal-check": "^1.1.8",
+        "openeuropa/behat-transformation-context": "~0.1",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9 || ^10",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
```

File: composer.json [hunk 4]
```diff
@@ -6,19 +6,22 @@
...
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpspec/prophecy-phpunit": "^2",
+        "phpunit/phpunit": "^7 || ^8 || ^9"
```
Comment (yenyasinn) [inline composer.json]: I'm not sure that PHPUnit 9 will be installed - we don't have additional dependencies that it is required. So has no sense to determine it here.

```suggestion
        "phpunit/phpunit": "^7 || ^8"
```

File: .drone.yml [hunk 5]
```diff
@@ -57,6 +57,19 @@ pipeline:
     commands:
       - ./vendor/bin/phpunit
 
+  test-drupal9-upgrade_status:
```
Comment (yenyasinn) [inline .drone.yml]: It was decided not to add D9 compatibility test steps to the drone.

File: composer.json [hunk 6]
```diff
@@ -6,19 +6,22 @@
...
-        "drush/drush": "^10.3",
-        "openeuropa/code-review": "~1.0",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6 || ~7"
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9 || ^10",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "^10.4",
+        "drupal/upgrade_status": "^3.4",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
```
We need this module locally during working on this task only. So no need to add it to the project.

File: composer.json [hunk 7]
```diff
@@ -6,19 +6,22 @@
...
+        "mglaman/drupal-check": "^1.1.8",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
```
The same as above

File: composer.json [hunk 8]
```diff
@@ -6,19 +6,22 @@
...
+        "openeuropa/behat-transformation-context": "~0.1",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9 || ^10",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpspec/prophecy-phpunit": "^2",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
```
Do we really need it?

File: .drone.yml [hunk 9]
```diff
@@ -57,6 +57,19 @@ pipeline:
     commands:
```
Comment (yenyasinn) [inline .drone.yml]: Please add to the `selenuim` settings

```
      - NODE_MAX_INSTANCES=5     
      - NODE_MAX_SESSION=5
```
See
https://github.com/openeuropa/oe_media/pull/163/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R34

File: composer.json [hunk 10]
```diff
@@ -6,19 +6,22 @@
...
-        "cweagans/composer-patches": "^1.7",
-        "drupal/core-composer-scaffold": "^9.0",
-        "drupal/drupal-extension": "~4.1",
-        "drush/drush": "^10.3",
-        "openeuropa/code-review": "~1.0",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6 || ~7"
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9 || ^10",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
          "drupal/core-composer-scaffold": "^8.9 || ^9",
```

**Comment by saidatom** [comment]
New PR #9

**Pull request closed without merging**